### PR TITLE
Us07 us21 new

### PR DESCRIPTION
**Pull request by Reiamulette**
A branch off of US06-US08 New

### CODE REVIEW

File: parse.py [hunk 1]
```diff
@@ -16,10 +16,21 @@ def get_individual_data_by_key(individuals, id, key):
...
+    #meta_data = None
+    #for meta in families[id]:
+    #    if meta[0] == key: meta_data = meta[1]
+    #return meta_data
+    family_data_list = families[id]
+    for tuple in family_data_list:
+        if tuple[0] == key:
+            return tuple[1]
+    #if execution reaches here key was not found
+    return false
```
Comment (chetan-plrch) [inline parse.py:92]: Same over here, false should be False right?

File: parse.py [hunk 2]
```diff
@@ -345,23 +361,103 @@ def check_birth_after_parent_marriage(families, individuals):
...
+            print("Age is over 150")
+    return is_valid    
+
+#US21
+def check_gender_role(families, individuals):
+    is_valid = True
+    for id in families:
+        husbID = get_family_data_by_key(families,id,"HUSB")
+        if husbID:
+            Gender = get_individual_data_by_key(individuals,husbID,"SEX")
```
Comment (chetan-plrch) [inline parse.py:511]: Can we follow variable naming convention, and change Gender to gender please?

**Review by chetan-plrch — CHANGES_REQUESTED** [review]

File: parse.py (hunk 1 above)
Comment (Reiamulette) [reply parse.py:92]: Addressed in US34-US35

File: parse.py (hunk 2 above)
Comment (Reiamulette) [reply parse.py:511]: Addressed in US34-US35

**Pull request closed without merging**